immuni_exposure_ingestion/helpers/risk_level.py

```diff
@@ -39,4 +41,8 @@ def extract_keys_with_risk_level_from_upload(upload: Upload) -> Iterable[Tempora
     for key in keys_at_risk:
         key.transmission_risk_level = TransmissionRiskLevel.highest
 
-    return keys_at_risk
+    return (
+        [key for key in keys_at_risk if key.expires_at < datetime.utcnow()]
```

Comment:
I'd add a TODO, or at least a NOTE, regarding handling—instead of discarding—current day TEKs.